src/meshapi/management/commands/replay_join_records.py

```diff
@@ -0,0 +1,75 @@
+from argparse import ArgumentParser
+from dataclasses import asdict, fields
+from datetime import datetime
+from typing import Any
+
+from django.core.management.base import BaseCommand
+from prettytable import PrettyTable
+
+from meshapi.util.join_records import JOIN_RECORD_PREFIX, JoinRecord, JoinRecordProcessor
+from meshapi.views.forms import JoinFormRequest, process_join_form
+
+
+class Command(BaseCommand):
+    help = "Replay join form submissions that we may not have accepted properly"
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "--noinput", action="store_true", help="Tells Django to NOT prompt the user for input of any kind."
+        )
+
+        parser.add_argument("--all", action="store_true", help="Fetch all Join Records, not just failed ones")
+
+        parser.add_argument("--look", action="store_true", help="Print Join Records and quit")
```

Comment:
nit: This might be safer as the inverse. i.e. a flag you pass in order to enable writes, not a flag to disable them

I see we have a confirmation prompt though so probably fine